### PR TITLE
MethylExtract recipe

### PR DESCRIPTION
**Pull request by phue**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

As this is my first recipe, I kindly ask for review
@bioconda/core

### CODE REVIEW

File: recipes/methylextract/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+about:
+  home: http://bioinfo2.ugr.es/MethylExtract/
+  license: GNU General Public License v3 (GPLv3)
+  license_family: GPL3
+  summary: High-Quality methylation maps and SNV calling from whole genome bisulfite sequencing data
+
+test:
+  commands:
+    - which MethylExtract.pl
```
Comment (bgruening) [inline recipes/methylextract/meta.yaml]: Can you run the scripts as well? With some `--version` or `--help` flag?

Comment (phue) [reply recipes/methylextract/meta.yaml]: Thanks for your review. 
Unfortunately, the scripts don't have such flags. And running them without arguments results in exitcode 255, which makes the test fail.

I guess another approach would be to grep for some string in the usage information that gets printed when calling the script, if you prefer that over 'which'.

**Comment by phue** [comment]
done :)

**Comment by bgruening** [comment]
Cool! Thanks a lot!

**Pull request merged**